frontend/src/components/chats/components/current-chat/styles.module.scss

```
@@ -15,7 +15,7 @@
 .currentChatContent {
   display: flex;
   flex: 1 1 auto;
-  flex-direction: column;
+  flex-direction: column-reverse;
```

```suggestion
  flex-direction: column;
```
do we do the sort on the backend?

---

sort is from the newest message

---

change the order. we need to handle this behaviour by the scripts

---

But here it does not depend on the order. If I use the column and column-reverse the order does not change. Here we get the expected result: messages list grows not from top to bottom, but from bottom to top